src/styles/components/about-us/flippingCard.module.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+    background-color: (var, --color-beige);
+    border-radius: 12px;
+}
```

Comment:
You can remove the container class and move these styles to `FlipCard` so there isn't the need of an extra container